docs/README.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+Kernel Module Management also known as KMM is a Kubernetes operator which provides a simple way to manage, build, sign or deploy out-of-tree kernel modules, drivers or device plugins on selected nodes of an existing Kubernetes cluster.
+
+Once installed we will get a Deployment object consisting of a ReplicaSet with a pod acting as a controller manager in the new Namespace kmm-operator-system. 
+
+This controller provides the use of a new Kubernetes API object called Module which sets if the driver image already exists and has to be loaded, if it has to be in-cluster built (a ConfigMap should be provided to get this), if it will use a device plugin and also on which nodes would be deployed as a new DaemonSet will be created for each node’s kernel version matching the settings in a DriverContainer or device plugin within the Module definition.
```

Comment:
```suggestion
This controller provides the use of a new Kubernetes API object called `Module` which sets if the driver image already exists and has to be loaded, if it has to be in-cluster built (a ConfigMap should be provided to get this), if it will use a device plugin and also on which nodes would be deployed as a new DaemonSet will be created for each node’s kernel version matching the settings in a DriverContainer or device plugin within the Module definition.
```